src/server/src/main/resources/db/cassandra/022_cluster_states.cql

```diff
@@ -0,0 +1,18 @@
+--
+--  Copyright 2019-2019 The Last Pickle Ltd
+--
+--  Licensed under the Apache License, Version 2.0 (the "License");
+--  you may not use this file except in compliance with the License.
+--  You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+--  Unless required by applicable law or agreed to in writing, software
+--  distributed under the License is distributed on an "AS IS" BASIS,
+--  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+--  See the License for the specific language governing permissions and
+--  limitations under the License.
+--
+-- Upgrade to store cluster state and last jmx connection time
+
+ALTER TABLE cluster ADD state text, last_contact date;
```

Comment:
Why not use a timestamp for `last_contact`? It would match the other storage backend types.

Reply:
the field will only store the day. (is there a sql equivalent?)

i may need to change it to timestamp though for compatibility across c* versions…